types/bcrypt/index.d.ts

```diff
@@ -121,7 +121,7 @@ export declare function hash(
     data: string | Buffer,
     saltOrRounds: string | number,
     callback: (err: Error | undefined, encrypted: string) => any,
```

Comment:
Reading the implementation, it seems like it only returns the promise when it doesn't receive a callback, so this doesn't seem right to me.

Reply:
Agreed; this should probably be an overload